docs/src/quickstart/index.md

```diff
@@ -0,0 +1,223 @@
+# {#ref-qs} Installation and Setup
+
+This quickstart guide outlines the step-by-step instructions for getting started with programming in Reach.
+You can install on the following OS systems:
+
+[Windows](##ref-qs-win)
+[Linux](##ref-qs-linux)
+[MacOS](##ref-qs-mac)
+
+# {#ref-qs-win} Windows
+
+Reach requires [make](https://en.wikipedia.org/wiki/Make_(software)), [Docker](https://www.docker.com/get-started), and [Docker Compose](https://docs.docker.com/compose/install/).
```

Comment:
This is a weird thing to say, because a Windows user can't really install Make and when they install Docker Desktop, it comes with Docker Compose. They first need to install WSL, then Docker, then they're good to go